app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
```

Comment:
Sorry, i forget abour this line
better not to change cls attribute through object even if this attribute is mutable
So fix this and all ok